plugins/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
 add_subdirectory(devices)
 add_subdirectory(instructions)
 add_subdirectory(traps)
+
+if(ENABLE_COMPLX)
```

Comment:
Honestly don't like this instead of adding this flag here why not ENABLE_BWLCD=OFF ENABLE_COLORLCD=OFF

Reply:
Agreed, updated that (removed that there, instead put `AND ENABLE_COMPLX` on those two toggles since they're useless without complx) in one of the previous commits